src/components/common/SendCode.vue

```diff
@@ -23,16 +35,43 @@ export default {
       type: Boolean,
       required: true,
     },
+    timeout: {
+      type: Number,
+      default: DEFAULT_SEND_CODE_TIMEOUT,
```

Comment:
if `DEFAULT_SEND_CODE_TIMEOUT` is used only in this place, no need to declare it in `constants` file